Gems/ROS2/Code/Source/Frame/ROS2FrameComponent.cpp

```diff
@@ -163,6 +198,17 @@ namespace ROS2
         return m_namespaceConfiguration.GetNamespace(parentNamespace);
     }
 
+    AZ::Name ROS2FrameComponent::GetJointName() const
+    {
+        return m_jointName;
+    }
+
+    void ROS2FrameComponent::SetJointName(const AZStd::string& jointNameString)
```

Comment:
@danielemorra98 It may be worth documenting what the intended input format of the jointNameString is. Looking at the code superficially, I believe it should be relative (in the sense of without namespace prefix) and passing in a string with namespace such as `/robot1/joint3` would lead to something like `/robot1//robot1/joint3`?

Reply:
It is relative. The absolute name (with namespace) is stored in a _AZ::Name_ variable. 
The `JointPublisherComponent` and `ManipulatorControllerComponent` retrieve the last in order to build the hierarchy map.

Reply:
Adding a comment suggestion to `ROS2FrameComponent.h`.